cmd/cmd.go

```diff
@@ -20,14 +21,15 @@ var (
 )
 
 func commands() []*cli.Command {
-	return []*cli.Command{
+	return append(
+		rootcmd.Commands(),
```

Comment:
Hva med `logincmd` i stedet for `rootcmd`? Flere av de andre ligger også på root.

Reply:
Forsøkte å følge strukturen litt, men føles veldig rart ut å ha en pakke for hver commando som ender på root, så tenkte å ha en gruppering for root. Alternativt tenker jeg å bare legge de rett i `cmd` mappa